e2e/support/commands/api/composite/createDashboardWithQuestions.js

```diff
@@ -10,9 +10,12 @@ Cypress.Commands.add(
           questions.map(query =>
             cy.createQuestionAndAddToDashboard(query, dashboard.id),
           ),
-        ).then(questions => {
+        ).then(dashcardResponses => {
```

Comment:
The result was null before, and now it's not null but an array of the dashcard response.